cmd/gitserver/main.go

```diff
@@ -496,7 +497,8 @@ func syncRateLimiters(ctx context.Context, store database.ExternalServiceStore,
 		if instanceCount > 0 {
 			break
 		}
-		log15.Warn("found zero gitserver instance, trying again in %s", backoff)
+
+		logger.With(log.Duration("backoff", backoff)).Warn("found zero gitserver instance, trying again in")
```

Comment:
```suggestion
		logger.Warn("found zero gitserver instance, trying again after backoff", log.Duration("backoff", backoff))
```

If the field is used only once, you can simply include it after the message - only use `With` if you are creating a sub-logger